examples/web-wagmi/src/misc/MiscPage.tsx

```diff
@@ -1,39 +1,53 @@
-import { LinkCard } from '../components/LinkCard';
+import { useActiveWallet } from '@lens-protocol/react';
 
-const miscHooks = [
-  {
-    label: 'useCurrencies',
-    description: `View all the ERC20 tokens supported by the Lens protocol.`,
-    path: '/misc/useCurrencies',
-  },
-  {
-    label: 'useNotifications',
-    description: `View notifications for the active profile.`,
-    path: '/misc/useNotifications',
-  },
-  {
-    label: 'useUnreadNotificationCount',
-    description: `View the total unread notification count for the active profile.`,
-    path: '/misc/useUnreadNotificationCount',
-  },
-  {
-    label: 'useApproveModule',
-    description: `Approve a certain amount of an ERC20 Token to be used by a address`,
-    path: '/misc/useApproveModule',
-  },
-  {
-    label: 'useEnabledModules',
-    description: `View all the modules supported by the Lens Protocol`,
-    path: '/misc/useEnabledModules',
-  },
-  {
-    label: 'useRecentTransactions',
-    description: `Example of listing + clearing recent completed transactions.`,
-    path: '/misc/useRecentTransactions',
-  },
-];
+import { LinkCard } from '../components/LinkCard';
 
 export function MiscPage() {
+  const activeWallet = useActiveWallet();
+
+  const miscHooks = [
+    {
+      label: 'useCurrencies',
+      description: `View all the ERC20 tokens supported by the Lens protocol.`,
+      path: '/misc/useCurrencies',
+    },
+    {
+      label: 'useNotifications',
+      description: `View notifications for the active profile.`,
+      path: '/misc/useNotifications',
+    },
+    {
+      label: 'useUnreadNotificationCount',
+      description: `View the total unread notification count for the active profile.`,
+      path: '/misc/useUnreadNotificationCount',
+    },
+    {
+      label: 'useApproveModule',
+      description: `Approve a certain amount of an ERC20 Token to be used by a address`,
+      path: '/misc/useApproveModule',
+    },
+    {
+      label: 'useEnabledModules',
+      description: `View all the modules supported by the Lens Protocol`,
+      path: '/misc/useEnabledModules',
+    },
+    {
+      label: 'useRecentTransactions',
+      description: `Example of listing + clearing recent completed transactions.`,
+      path: '/misc/useRecentTransactions',
+    },
+
+    ...(activeWallet.data === null
```

Comment:
if you allow visiting this page as a logged-in user then you don't need this condition here. it is the example app so hiding features from users don't make much sense. they should be able to explore as much as possible.